lib/vsc/administration/slurm/sync.py

```diff
@@ -267,7 +267,7 @@ def slurm_institute_accounts(slurm_account_info, clusters):
     return commands
 
 
-def slurm_vo_accounts(account_page_vos, slurm_account_info, clusters):
+def slurm_vo_accounts(account_page_vos, slurm_account_info, clusters, host_institute=GENT):
```

Comment:
make `host_institute` positional arg

Reply:
@stdweird so you do not want a default value for `host_institute`?
should I do the same for `slurm_institute_accounts()`?

Reply:
the default in the options is enough
i don't understand the 2nd question

Reply:
see https://github.com/hpcugent/vsc-administration/pull/104/commits/7f0c787575efddc3ba2f014b4e563971fd6661e7